profile.php

```diff
@@ -214,44 +214,13 @@
     $mailaddress = "n/a";
     $mailpic = "";
 } else {
-    if ($player['Joined'] != NULL) { 
-        $joined = date("H:i d m y", $player['Joined']); 
+    if ($player['Joined'] != NULL) {
+        $joined = date("H:i d m y", $player['Joined']);
     } else {
-        $joined ="00:00 06 03 08"; 
+        $joined ="00:00 06 03 08";
     }
 
-    // Read the mail from the db and make it spambotsafe...
-    $mailaddress = $player['mail'];
-    $mailaddress = str_replace("@", " (at) ", $mailaddress);
-    $mailaddress = str_replace(".", " (dot) ", $mailaddress);
-    $jabbername = $player['Jabber'];
-    $jabbername = str_replace("@", " (at) ", $jabbername);
-    $jabbername = str_replace(".", " (dot) ", $jabbername);
-    $jabberpic = "<img border='1' src='images/jabber.gif' align='absmiddle' alt='Jabber' />";
-    $mailpic = "<img border='1' src='images/mail.gif' align='absmiddle' alt='email' /></a>";
-}
-if ($player['icq'] == "n/a") {
-    $icqnumber = "n/a";
-    $icqpic = "";
-} else {
-    $icqnumber = $player['icq'];
-    $icqpic = "<img border='1' src='images/icq.gif' align='absmiddle' alt='icq' />";
-}
-if ($player['aim'] == "n/a") {
-    $aimname = "n/a";
-    $aimpic = "";
-} else {
-    $aimname = $player['aim'];
-    $aimpic = "<img border='1' src='images/aim.gif' align='absmiddle' alt='aim' />";
-}
-if ($player['msn'] == "n/a") {
-    $msnname = "n/a";
-    $msnpic = "";
-} else {
-    $msnname = $player['msn'];
-    $msnname = str_replace("@", " (at) ", $msnname);
-    $msnname = str_replace(".", " (dot) ", $msnname);
-    $msnpic = "<img border='1' src='images/msn.gif' align='absmiddle' alt='msn' /></a>";
+
```

Comment:
Do we really need to delete it? Maybe just hide from public (not logged-in) viewers?